src/utilities.ts

```diff
@@ -1,24 +1,28 @@
+import { OrderItem } from "./types";
+
 // if the comma or decimal portions of the price need internationalization
 // then considering switching to numbro or formatjs
-function priceFormat(amountInCents, currencySymbol) {
+function priceFormat(amountInCents: number, currencySymbol?: string) {
   currencySymbol = currencySymbol || '$';
   return currencySymbol + (amountInCents / 100).toFixed(2).replace(/\B(?=(\d{3})+(?!\d))/g, ',');
 }
 
-function discountable(amountInCents, percentOff, amountOffInCents) {
-  var discount, appliedDiscount;
+export function discountable(
+  amountInCents: number, 
+  percentOff?: number, 
+  amountOffInCents?: number
```

Comment:
the tests seem to indicate a `null` type for these arguments, i replaced them with optional fields which accept `undefined` types.